src/service/position-observer/position-observer-impl.js

```diff
@@ -18,11 +18,10 @@ import {registerServiceBuilderForDoc} from '../../service';
 import {Services} from '../../services';
 import {dev} from '../../log';
 import {debounce} from '../../utils/rate-limit';
-import {PositionObserverEntry} from './position-observer-entry';
 import {
-  PositionObserverFidelity,
-  LOW_FIDELITY_FRAME_COUNT,
-} from './position-observer-fidelity';
+  PositionObserverEntry,
+  /* eslint no-unused-vars: 0 */ PositionObserverFidelity,
```

Comment:
Nit: Appending `Def` to the name will skip the lint error.

Reply:
but the value is an enum not a Def 😕  Should i do that.

Reply:
Oh, this is just being used as a typedef. Just use the `./position-observer-entry.PositionObserverFidelity` types instead.

Reply:
That's the first thing I try. 
Get 
```
Bad type annotation. Unknown type module$src$service$position_observer$position_observer_entry.PositionObserverFidelity
```
Strange that `./position-observer-entry.PositionInViewportEntryDef` works but `./position-observer-entry.PositionObserverFidelity` doesn't.

Reply:
That's likely during the extension compiling. What's the full output?

Reply:
```
src/service/position-observer/position-observer-impl.js:73: ERROR - Bad type annotation. Unknown type module$src$service$position_observer$position_observer_entry.PositionObserverFidelity
   * @param {!./position-observer-entry.PositionObserverFidelity} fidelity
```

get this running `gulp check-types`